cmake/Deps_Unix.cmake

```diff
@@ -79,30 +79,25 @@ endif ()
 if (WITH_ZIM)
     if (APPLE)
         # ICU from homebrew is "key-only", we need to manually prioritize it -> see `brew info icu4c`
-        set(ENV{PKG_CONFIG_PATH} "$ENV{PKG_CONFIG_PATH}:/usr/local/opt/icu4c@76/lib/pkgconfig:/opt/homebrew/opt/icu4c@76/lib/pkgconfig:/usr/local/opt/icu4c/lib/pkgconfig:/opt/homebrew/opt/icu4c/lib/pkgconfig")
+        # And we needs to find the correct one if multiple versions co exists.
+        set(ENV{PATH} "$ENV{PATH}:/usr/local/bin/:/opt/homebrew/bin") # add brew command into PATH
```

Comment:
This line is needed for local IDEs, because GUI apps launched via icon doesn't inherit the `PATH` from the login shell's .bashrc or .zshrc.